src/getClasses.js

```diff
@@ -6,29 +6,22 @@
  */
 export function getClasses( el )
 {
-  let classNames;
-
-  try
+  if( !el.hasAttribute( 'class' ) )
   {
-    classNames = el.classList.toString().split( ' ' );
+    return [];
   }
-  catch ( e )
-  {
-    if( !el.hasAttribute( 'class' ) )
-    {
-      return [];
-    }
 
+  try {
+    return Array.from(el.classList);
```

Comment:
No, it will not take care actually, there is not `babel-polyfill`. Good catch! @ronkorving 
@jeetiss Can you pls change it to `slice` ?